README.md

```diff
@@ -36,6 +36,14 @@ $ npm install --save eventemitter3        # npm
 $ component install primus/eventemitter3  # Component
 $ bower install eventemitter3             # Bower
 ```
+## CDN
+
+Recommended CDNs:
+
+```text
+https://cdnjs.cloudflare.com/ajax/libs/eventemitter3/2.0.2/index.min.js
```

Comment:
Should we remove this? There is no UMD bundle and unpkg uses cloudflare so maybe it's better to only keep unpkg.